policy/release/olm.rego

```diff
@@ -122,6 +122,34 @@ deny contains result if {
 	result := lib.result_helper(rego.metadata.chain(), [error])
 }
 
+# METADATA
+# title: Unpinned images in input snapshot
```

Comment:
Since ec-cli always resolves image references to a digest, this is more like a validation on the ApplicationSnapshot itself, is that correct? Maybe it belongs on a different package?

Reply:
well, it looks like ec-cli does resolve the input component but not the entire snapshot..unless I am missing something.

Also, we need this check to run for operator bundles because that's the one using snapshot components later to verify if it matches with relatedImages. Is there any other package more suitable for this?

Reply:
> well, it looks like ec-cli does resolve the input component but not the entire snapshot..unless I am missing something.

Yes. That's what I meant to by my statement above. Maybe not as clear :smile: 

> Also, we need this check to run for operator bundles because that's the one using snapshot components later to verify if it matches with relatedImages. Is there any other package more suitable for this?

Understood. I'm just thinking that a policy that verifies ApplicationSnapshots always contain pinned image references is useful in all cases, not just for OLM.

Let's keep it here for now. We can refactor later.